space_age/.byebug_history

```diff
@@ -1,196 +0,0 @@
-exit
-admin = User.create(username: "adminor", email: "emailzo", password: "password", password_confirmation: "password", role: 1)
-User.all
-exit
-total.to_i
-total = one.price + two.price
-one.price + two.price
-one.price
-two = Package.find(1074)
-one = Package.find(1073)
-Package.all
-OrderPackage.all.count
-Order.package.all.count
-Package.all.count
-Order.all.count
-Order.all
-current_user
-exit
-current_user
-exit
-current_user == Order.find(params[:id]).user
-Order.find(params[:id]).user
-params
-current_user
-exit
-current_user
-exit
-o.user
-o = Order.find(246)
-o = _
-Order.find(246)
-params
-exit
-User.find(794)
-params
-exit
-Order.find(165)
-Orders.find(165)
-params
-current_user
-exit
-params
-current_user
-exit
-User.find(503)
-params
-exit
-params
-current_user
-exit
-current_user.id.to_s == params[:id]
-current_user.id.to_s
-current_user.id
-params[:id]
-params
-current_user
-current_user == User.find(params[:id])
-exit
-User.find(params[:id])
-params
-current_user
-exit
-User.all
-current_user
-exit
-current_user
-User.find(233)
-Order.all
-params
-@order = Order.find(params[:id])
-exit
-@order
-exit
-User.find(177)
-params
-current_user
-exit
-User.find(813)
-current_user
-params
-Order.find(params[:id])
-exit
-@order
-exit
-Order.find(537)
-current_user
-User.find(537)
-params
-exit
-User.find(455)
-params
-exit
-current_user
-exit
-current_user
-exit
-current_user
-exit
-user2.id
-user2
-exit
-Destination.find_by(planet: "neptune")
-params[:planet]
-params
-exit
-params
-exit
-params
-exit
-@user
-exit
-user
-admin
-exit
-user
-admin
-exit
-User.find(params[:id])
-current_user
-params
-exit
-User.find(206)
-session[:user_id]
-session
-@user
-rails c
-admin
-current_user
-exit
-current_user
-params
-exit
-current_user
-params
-exit
-user
-exit
-user
-admin
-exit
-current_user
-next
-session[:user_id]
-session
-exit
-current_user
-exit
-params_check
-@user
-next
-exit
-@user.update(username:"j")
-@user
-exit
-@user.errors
-@user.erros
-@user.update(username:"J").errors
-@user.update(username:"J")
-@user
-exit
-step
-@user
-exit
-package4
-package1
-exit
-package1.destination
-package1.destinatino
-package4.destination
-package4
-exit
-package4
-package3
-package2
-package1
-exit
-x.to_i
-x
-exit
-x.to_i
-x
-exit
-Package.find("255").price
-Package.find("255")
-package
-exit
-package
-exit
-Package.find(id).price * 2
-Package.find(id).price
-Package.find(id)
-Package.all
-exit
-Package.all
-Package.find("1")
```

Comment:
let's be sure and look in to putting this into the .gitignore so we don't have to worry about it.